docs/Import-the-Seed-Database.md

```diff
@@ -4,7 +4,7 @@ The next step is to populate your cBioPortal instance with all the required back
 
 ## Download the cBioPortal Database
 
-A cBioPortal seed database can be found on [the datahub page](https://github.com/cbioportal/datahub/blob/master/seedDB/README.md).
+A cBioPortal seed database for human can be found on [the datahub page](https://github.com/cBioPortal/datahub/blob/84fd66daf8325ad9721895d1cc503653686de15e/seedDB/README.md). If you are looking for mouse, check [this link](https://github.com/cBioPortal/datahub/blob/b02ac7037c8febeb86d6635602fc98327fb77c50/seedDB_mouse/README.md).
```

Comment:
Yes, it would make things a bit easier: when you release a new version of the seed database on datahub, you'd then have to change the documentation in the same folder to list the minimum cBioPortal version, instead of having to wait for a release of cBioPortal that uses the new seed release.